src/CMakeLists.txt

```diff
@@ -11,46 +11,46 @@ set(BUILD_WSL false CACHE BOOL "Enable for builds on WSL")
 include_directories(${PROJECT_SOURCE_DIR}/src ${EXT_PROJECTS_DIR}/easyloggingpp-9.97.0)
 
 file(GLOB SRC_FILES *.cpp
-        Core/*.cpp 
-        Core/Config/*.cpp 
-        Core/Index/*.cpp 
-        Events/*.cpp 
-        Events/Population/*.cpp 
-        Helpers/*.cpp 
+        Core/*.cpp
```

Comment:
Still, you don't have to pull request this file

Reply:
I haven't done anything here. It is the same as your old file.

Reply:
I really don't know how to choose files, all files from my local get commited to github. I am not sure how to choose and remove them for pull request.

Reply:
You have to find a way to not commit this file.